app/models/concerns/host_common.rb

```diff
@@ -204,6 +204,24 @@ def available_puppetclasses
     environment.puppetclasses - parent_classes
   end
 
+  def inherited_params(include_source = false)
```

Comment:
This method is only used in Host::Managed, a separate one is used for Hostgroup, so it probably shouldn't be in HostCommon - or there needs to be some unification.  I think it's probably better to move it back, since inheritance of parameters through hosts and host groups is different.